### PR TITLE
KLIMA-77: linting and img/skeleton-component

### PR DESCRIPTION
**Pull request by sinejespersen**
#### Link to ticket

https://jira.itkdev.dk/browse/KLIMA-77

#### Description

- added some linting of react
- added skeleton component
- changed div -> img ^because of onload for skeleton component

#### Screenshot of the result

(looks more smooth in real life, github will not allow my large gifs)

skeleton

### CODE REVIEW

File: assets/react/components/grid-item.jsx [hunk 1]
```diff
@@ -54,9 +96,38 @@ const ItemIcon = styled.img`
...
+
+const ContentWrapper = styled.div`
+  position: relative;
+`;
+
+const Item = styled.img`
+  object-fit: cover;
   position: relative;
-  background-image: var(--background-image);
   background-position: center;
```
Comment (yepzdk) [inline assets/react/components/grid-item.jsx]: Are bg image and position still relevant?

File: README.md [hunk 2]
```diff
@@ -69,9 +79,10 @@ Following parameters is possible to configure from the `screen` -> `variant` fie
 | ctaBoxDescription     | Text          | `"ctaBoxDescription":"Skan koden og indsend dit bidrag"` |
 | ctaBoxImage           | Url           | `"https://pathtoimagesource.io/ctaimage.svg"` |
 | ctaBoxBackgroundColor | Hex format    | `"ctaBoxBackgroundColor":"#fff"` |
+| colors                | array of hex  | `["#ee7752", "#e73c7e", "#23a6d5", "#23d5ab"]` |
```
Comment (yepzdk) [inline README.md]: colors seem a bit broad as a name in this context? If we only use them in relation to the skeleton, maybe we could make this more obvious with a name like skeletonColors?

**Review by yepzdk — APPROVED** [review]
A couple of non blocking comments for your consideration.
Looks very good! And nice addition of linting.

File: assets/react/components/grid-item.jsx (hunk 1 above)
Comment (sinejespersen) [reply assets/react/components/grid-item.jsx]: prob not, will remove

File: README.md (hunk 2 above)
Comment (sinejespersen) [reply README.md]: will change this too :)

**Pull request merged**